UI/GroceyStore_UI/grocery_store/src/Footer.js

```diff
@@ -1,22 +1,27 @@
-import React, {Component} from "react"
-import {Router, Link} from 'react-router-dom'
+import React, { Component } from "react"
+import { Router, Link } from 'react-router-dom'
 
-class Footer extends Component{
+class Footer extends Component {
     constructor(props) {
         super(props);
         console.log(this.props)
-    }    
+    }
 
-    render(){
-        return(
+    render() {
+        return (
             <div id="footer">
                 <Link className="footerLink" to="/userprofile"> User Profile</Link>
                 <Link className="footerLink" to="/cart"> Cart</Link>
                 <Link className="footerLink" to="/history"> History</Link>
                 <Link className="footerLink" to="/Notification"> Notifaction</Link>

```

Comment:
I think this is a merge issue, we need to delete these lines. The lines below this block are the updated ones